### DISCUSSION TITLE
Transmission on startup

### DISCUSSION THREAD

**Discussion by Baldie274** [transmission/transmission#4533] [Q&A]
Hi! I'm using Xfce and when I installed Transmission, it keeps launching as a startup application. How do I disable that? Can't do it using Xfce GUI, as Settings-Session and Startup doesn't list Transmission there. There is also no way to do it in Transmission GUI itself. Maybe I can edit a config file? Any hints? Thank you!

**Comment by ckerr** [comment]
I don't remember anything in the code that would make it a startup application, so my first guess is that it's something in the packaging?

So I might check out Xfce's [startup applications config](https://unix.stackexchange.com/questions/669372/xfce4-session-and-startup-where-are-autostart-items-saved) and see if Transmission is showing up there on your system
